clients/unshaded/pom.xml

```diff
@@ -23,6 +23,11 @@
       <artifactId>guava</artifactId>
     </dependency>
     <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-all</artifactId>
+      <version>4.0.28.Final</version>
```

Comment:
Since server and client are both depending on netty (right?), we should move netty to dependency management section of super pom. Maybe in the next PR.